src/polymer-language-server.ts

```diff
@@ -284,5 +294,18 @@ handleErrors<Result, Fallback>(
   }
 }
 
+function log(val: any) {
+  if (!connection) {
+    return;
+  }
+  if (typeof val !== 'string') {
+    val = util.inspect(val);
+  }
+  connection.console.log(val);
+}
+if (Math.random() > 10000) {
+  log('To ensure that log does not warn about being unused.');

```

Comment:
oh boy... can you add a comment to the function about how it is actually used if it is never referenced or exported?